src/components/WindowDimensions/WindowDimensions.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React, { Component } from 'react';
```

Comment:
Getting a `'React' is defined but never used  no-unused-vars` warning from this

Reply:
Right, makes sense... I wonder if our CI should fail when there are lint warnings.

Reply:
Not too sure of CI conventions but my instinct says it should fail on warning, since the point is to catch things before they get into the codebase.

Reply:
Or perhaps it be an error instead of a warning.